pages/api/upload_album.py

```
@@ -1221,20 +1218,144 @@ def get_folder_info_by_name():
         )
     except Exception as e:
-        print(f"Error starting server: {e}")
-        import traceback
-        print(traceback.format_exc())
+        print(f"Server error: {str(e)}")
+        sys.exit(1)
+
+# Add static route
+@app.route('/static/<path:filename>')
+def serve_static(filename):
+    print(f"=== Debug serve_static ===")
```

Ruff (0.7.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)